modules/upgrade/pages/server-update.adoc

```diff
@@ -31,27 +32,17 @@ zypper list-patches
 zypper patch
 ----
 
-
-
-.Procedure: Upgrading the Database Schema
-
-If a patch includes a new database schema upgrade, you will need to use the [command]``spacewalk-schema-upgrade`` command.
-
-. On the {productname} Server, at the command prompt, as root, ensure the database service is running:
-+
-----
-systemctl start postgresql
-----
-. Perform the schema update:
-+
-----
-spacewalk-schema-upgrade
-----
 . Restart the spacewalk services:
 +
 ----
 spacewalk-service start
 ----
++
+[NOTE]
+====
+[command]``spacewalk-schema-upgrade`` is not needed anymore.
+It will be run during [command]``spacewalk-service start`` automatically.
+====
```

Comment:
Should we add a reference to the Troubleshooting section in case the start/schema-migration fail?

Reply:
ok, makes sense.